python/mxnet/ndarray/ndarray.py

```diff
@@ -673,6 +686,100 @@ def reshape(self, shape):
         return NDArray(handle=handle, writable=self.writable)
 
+    def flatten(self, **kwargs):
+        """Convenience fluent method for :py:func:`ndarray.flatten`."""
+        return op.flatten(self, **kwargs)
+
+    def expand_dims(self, **kwargs):
+        """Convenience fluent method for ndarray.expand_dims.
+        See expand_dims"""
+        return op.expand_dims(self, **kwargs)
+
+    def tile(self, **kwargs):
+        """Convenience fluent method for ndarray.tile.
+        See tile"""
+        return op.tile(self, **kwargs)
+
+    def transpose(self, **kwargs):
+        """Convenience fluent method for ndarray.transpose.
+        See transpose"""
+        return op.transpose(self, **kwargs)
+
+    def flip(self, **kwargs):
+        """Convenience fluent method for ndarray.flip.
+        See flip"""
+        return op.flip(self, **kwargs)
+
+    def sum(self, **kwargs):
+        """Convenience fluent method for ndarray.sum.
+        See sum"""
+        return op.sum(self, **kwargs)
+
+    def nansum(self, **kwargs):
+        """Convenience fluent method for ndarray.nansum.
+        See nansum"""
+        return op.nansum(self, **kwargs)
+
+    def prod(self, **kwargs):
+        """Convenience fluent method for ndarray.prod.
+        See prod"""
+        return op.prod(self, **kwargs)
+
+    def nanprod(self, **kwargs):
+        """Convenience fluent method for ndarray.nanprod.
+        See nanprod"""
+        return op.nanprod(self, **kwargs)
+
+    def mean(self, **kwargs):
+        """Convenience fluent method for ndarray.mean.
+        See mean"""
+        return op.mean(self, **kwargs)
+
+    def max(self, **kwargs):
+        """Convenience fluent method for ndarray.max.
+        See max"""
+        return op.max(self, **kwargs)
+
+    def min(self, **kwargs):
+        """Convenience fluent method for ndarray.min.
+        See min"""
+        return op.min(self, **kwargs)
+
+    def norm(self, **kwargs):
+        """Convenience fluent method for ndarray.norm.
+        See norm"""
+        return op.norm(self, **kwargs)
+
+    def round(self, **kwargs):
+        """Convenience fluent method for ndarray.round.
+        See round"""
+        return op.round(self, **kwargs)
+
+    def rint(self, **kwargs):
+        """Convenience fluent method for ndarray.rint.
+        See rint"""
+        return op.rint(self, **kwargs)
+
+    def fix(self, **kwargs):
+        """Convenience fluent method for ndarray.fix.
+        See fix"""
+        return op.fix(self, **kwargs)
+
+    def floor(self, **kwargs):
+        """Convenience fluent method for ndarray.floor.
+        See floor"""
+        return op.floor(self, **kwargs)
+
+    def ceil(self, **kwargs):
+        """Convenience fluent method for ndarray.ceil.
+        See ceil"""
+        return op.ceil(self, **kwargs)
+
+    def trunc(self, **kwargs):
+        """Convenience fluent method for ndarray.trunc.
+        See trunc"""
+        return op.trunc(self, **kwargs)
+
```

Comment:
Do we need to support `repeat` as well?

Reply:
I was limiting the methods to unary ops with 0 or 1 argument before. Let me revisit the op list and see what else can be added.